courses/machine_learning/deepdive2/feature_engineering/solutions/3_Basic_Feature_Engineering_in_Keras.ipynb

```diff
@@ -0,0 +1,2007 @@
+{
```

Comment:
<p>I think there is a typo here?</p><p>the <code>append</code>  command got split by the <code>print</code>  line?</p><p>So, it should be?</p><p> </p><p>feature_columns = []</p><p>for header in numeric_cols:</p><p>&nbsp;&nbsp;scal_input_fn = get_scal(header)</p><p>&nbsp;&nbsp;feature_columns.append(feature_column.numeric_column(header, normalizer_fn=scal_input_fn))</p><p>  print('Total number of feature coLumns: ', len(feature_columns))</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>Yes, copy/paste error when I changed the variable num_c to numberic_cols.  Should be:  </p><p>feature_columns = []</p><p>for header in numeric_cols:</p><p>&nbsp;&nbsp;scal_input_fn = get_scal(header)</p><p>&nbsp;&nbsp;feature_columns.append(feature_column.numeric_column(header, normalizer_fn=scal_input_fn))</p>